src/cloudant/client.py

```diff
@@ -55,11 +55,11 @@ class CouchDB(dict):
 
     def __init__(self, user, auth_token, admin_party=False, **kwargs):
         super(CouchDB, self).__init__()
-        self._cloudant_user = user
-        self._cloudant_token = auth_token
-        self._cloudant_session = None
-        self.cloudant_url = kwargs.get('url')
-        self._cloudant_user_header = None
+        self._user = user
+        self._auth_token = auth_token
+        self._client_session = None
+        self.client_url = kwargs.get('url')
```

Comment:
I suppose it depends on how you look at it.  I named it `client_url` because the url attribute continues to come up as you move down the object management chain.  As in, for a database object you would have a `database_url` and for a document object you would have a `document_url`.  So I was following the naming convention pattern in naming it `client_url`.  I think it may be odd to have a public attribute on a client object named `server_url` but I could be convinced otherwise since I do see your point.  After knowing all this, do you still think it should be `server_url`?  Perhaps @brynh can chime in as well.